packages/plugin-core/src/features/BacklinksTreeDataProvider.ts

```diff
@@ -6,8 +6,11 @@ import {
   FoundRefT,
   sortPaths,
 } from "../utils/md";
-import _ from "lodash";
-import { ICONS } from "../constants";
+import _, { Dictionary } from "lodash";
+import { DendronContext, ICONS } from "../constants";
+import { Logger } from "../logger";
+import { VSCodeUtils } from "../vsCodeUtils";
+import { BacklinkSortOrder } from "../types";
```

Comment:
When updating the sort order, can we indicate via the label what the current sort order is?

```
Linked. 3 link(s). Sorted by ...
```

Reply:
also, can we also show last updated time in the Backlink?

```
* Some bullet wiht [[link]] on line 18, updated: 2021-12-12 
```

Reply:
> When updating the sort order, can we indicate via the label what the current sort order is?

Not clear which label you are referring to. I don't see any high level labels about backlinks. 
----------------------------------------------------------------------------------------------------

Tried the suggested UX of adding the updated time on the line with `on line 18`. However, when we have multiple links from the same note we just end up duplicating the same data, which makes it for a bit of cluttered UX.

<img width="667" alt="Screen Shot 2021-12-22 at 5 08 36 PM" src="https://user-images.githubusercontent.com/4050134/147076646-42edc8b2-73c2-4971-a9d8-5ccd5b22d13b.png">

So approaching it from adding the update time on the group level to avoid the clutter.

<img width="590" alt="Screen Shot 2021-12-22 at 5 11 50 PM" src="https://user-images.githubusercontent.com/4050134/147076649-5ae3aed2-72ac-4a84-98f2-fe69e8a44d86.png">

Also prefixing the `updated` with `note updated`. Otherwise, it becomes ambiguous whether the links particular were updated or the note was updated. 

Also for date formatting using the shorthand month wording to avoid ambiguity between different date formattings.

Reply:
> Not clear which label you are referring to. I don't see any high level labels about backlinks.

I'm talking about the `Linked` label which you updated to show updated time. might not be ideal since the sort order will show in every note so feel free to disregard

Reply:
Yea, adding the sort order into each label is likely going to be quite cluttery, so would rather not. 

We do have the sort order mentioned in the hover over label right now, that should at least give some indication to the user, if they are curious.